homeassistant/components/qbittorrent/coordinator.py

```diff
@@ -31,8 +36,15 @@ def __init__(self, hass: HomeAssistant, client: Client) -> None:
             update_interval=timedelta(seconds=30),
         )
 
-    async def _async_update_data(self) -> dict[str, Any]:
+    def update(self) -> SessionStats:
```

Comment:
Where does this object come from

Reply:
I also dont think these changes make a big difference

Reply:
lol thats a great question

Reply:
It was a bigger deal we were grabbing those extra APIs, I can change it back